pyuploadcare/api_resources.py

```diff
@@ -231,12 +231,28 @@ def store(self):
         """
         self._info_cache = rest_request('PUT', self._api_storage_uri)
 
-    def copy(self, effects=None, target=None):
+    def copy(self, effects=None, target=None, make_public=None, store=None, pattern=None):
         """Creates File copy
 
-        If ``target`` is ``None``, copy file to Uploadcare storage otherwise
-        copy to target associated with project.
-        Add ``effects`` to ``self.default_effects`` if any.
+      - If ``target`` is ``None``, copy file to Uploadcare storage otherwise
+        copy to target associated with project;
+      -  Add ``effects`` to ``self.default_effects`` if any;
+      -  To forbid a public from accessing your files on the target storage set
+        ``make_public`` option to be ``false``,
+        relevant when ``target`` has a value set.
+      - Specify  ``pattern`` option to set the file name
```

Comment:
currently we're working with S3, so: http://docs.aws.amazon.com/AmazonS3/latest/dev/UsingMetadata.html

also please use English in public repo :)